test/models/user_test.rb

```diff
@@ -8,8 +8,53 @@ def setup
     @user = create(:user)
   end
 
-  test "`require_2sv` defaults to false" do
-    refute build(:user).require_2sv
+  context "#require_2sv" do
+    should "default to false for normal users" do
+      refute create(:user).require_2sv
```

Comment:
The same applies here and elsewhere in this test where the predicate form is not used. Stylistic nit-pick, soz.